yellowbrick/features/rankd.py

```diff
@@ -205,12 +240,12 @@ def transform(self, X, **kwargs):
 
         Returns
         -------
-        Xp : ndarray
+        X' : ndarray
             The transformed matrix, X'
         """
         # Rank and draw the input matrix
-        Xp = self.rank(X)
-        self.draw(Xp, **kwargs)
+        R = self.rank(X)
+        self.draw(R, **kwargs)
```

Comment:
Yes, my aim here was to add clarity.  The docstring in `transform` specified that `Xp` was the method's return value, but `Xp` actually referred to the rank matrix in the code, and that variable is not returned.  On the other hand the docstring for the `rank` method referred to the same rank matrix as `R`, so I thought it was best to keep consistent and call it the same thing everywhere.  

In any case, I think your explanations will help mitigate confusion in the future.  I've added them in.